apps/landing_partner/src/app/globals.css

```
@@ -1,26 +1,27 @@
+@font-face {
+  font-family: 'Pretendard';
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)